src/twisted/spread/pb.py

```diff
@@ -455,9 +491,11 @@ class CopiedFailure(RemoteCopy, failure.Failure):
     def printTraceback(self, file=None, elideFrameworkCode=0, detail='default'):
         if file is None:
             file = log.logfile
+        failureType = self.type
+        if not isinstance(failureType, str):
+            failureType = failureType.decode("utf-8")
         file.write("Traceback from remote host -- ")
-        file.write(self.traceback)
-        file.write(self.type + ": " + self.value)
+        file.write(failureType + ": " + self.value)
```

Comment:
Having just decoded `failureType` to unicode, wouldn't it make most sense to concatenate it with unicode rather than a native string?